README.md

```diff
@@ -1,16 +1,148 @@
 # OpenCensus Specs
 
-## General Design
-* For details about the library structured see [Namespace and Package](https://github.com/census-instrumentation/opencensus-specs/blob/master/NamespaceAndPackage.md)
+This is a high level design of the OpenCensus library, some of the API examples may be written 
+(linked) in C++, Java or Go but every language should translate/modify them based on language 
+specific patterns/idioms. Our goal is that all libraries have a consistent "look and feel".
 
-## Trace Design
-* Trace API is described [here](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/README.md)
+This repository uses terminology (MUST, SHOULD, etc) from [RFC 2119][RFC2119].
 
-* Data model is defined [here](https://github.com/census-instrumentation/opencensus-proto/blob/master/trace/trace.proto)
+## Overview
+Today, distributed tracing systems and stats collection tend to use unique protocols and 
+specifications for propagating context and sending data to backend processing systems. This 
+is true amongst all the large vendors, and we aim to standardize these APIs and data models.
```

Comment:
"we aim to standardize these APIs and data models."
is our aim to standardize? or is our aim to provide a reliable implementations in service of frameworks and agents. We do that by standardizing APIs and data models.

I think this leads to the next paragraph better

Reply:
Done.